src/coreclr/src/vm/stubmgr.h

```diff
@@ -227,7 +227,7 @@ class StubManager
     static BOOL TraceStub(PCODE stubAddress, TraceDestination *trace);
 
     // if 'trace' indicates TRACE_STUB, keep calling TraceStub on 'trace', until you get out of all stubs
-    // returns true if successfull
+    // returns true if successful
```

Comment:
You didn't actually follow the caps/period pattern here. (And as mentioned, I prefer it as it is)